plugins/module_utils/aci.py

```diff
@@ -81,7 +83,7 @@ def aci_argument_spec():
     return dict(
         host=dict(type='str', required=True, aliases=['hostname'], fallback=(env_fallback, ['ACI_HOST'])),
         port=dict(type='int', required=False, fallback=(env_fallback, ['ACI_PORT'])),
-        username=dict(type='str', default='admin', aliases=['user'], fallback=(env_fallback, ['ACI_USERNAME', 'ANSIBLE_NET_USERNAME'])),
+        username=dict(type='str', required=False, aliases=['user'], fallback=(env_fallback, ['ACI_USERNAME', 'ANSIBLE_NET_USERNAME'])),
```

Comment:
could this impact current users, which leverage admin default value?

Reply:
I think the default value is utilized in two other places in our code which is taken care of. However, we'll need to look into this.